core/db/migrate/20160511072249_change_collation_for_tag_names.rb

```diff
@@ -0,0 +1,9 @@
+# This migration is added to circumvent issue #623 and have special characters
+# work properly
+class ChangeCollationForTagNames < ActiveRecord::Migration
+  def up
+    if ActsAsTaggableOn::Utils.using_mysql?
+      execute("ALTER TABLE spree_tags MODIFY name varchar(255) CHARACTER SET utf8 COLLATE utf8_bin;")
```

Comment:
This SQL causes DB fail on MySQL https://travis-ci.org/spree/spree_auth_devise/jobs/133343113